{
  "about": "https://jvndb.jvn.jp/ja/contents/2020/JVNDB-2020-006108.html",
  "title": "GitHub Enterprise Server における外部からアクセス可能なファイルまたはディレクトリに関する脆弱性",
  "link": "https://jvndb.jvn.jp/ja/contents/2020/JVNDB-2020-006108.html",
  "description": "GitHub Enterprise Server には、外部からアクセス可能なファイルまたはディレクトリに関する脆弱性が存在します。",
  "identifier": "JVNDB-2020-006108",
  "references": [
    {
      "text": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10516",
      "source": "CVE",
      "id": "CVE-2020-10516"
    },
    {
      "text": "https://nvd.nist.gov/vuln/detail/CVE-2020-10516",
      "source": "NVD",
      "id": "CVE-2020-10516"
    },
    {
      "text": "https://cwe.mitre.org/data/definitions/552.html",
      "id": "CWE-552",
      "title": "外部からアクセス可能なファイルまたはディレクトリ(CWE-552)"
    }
  ],
  "cpe": [
    {
      "text": "cpe:/a:github:github_enterprise",
      "version": "2.2",
      "vendor": "GitHub",
      "product": "GitHub Enterprise"
    }
  ],
  "cvss": [
    {
      "version": "2.0",
      "score": "7.5",
      "type": "Base",
      "severity": "High",
      "vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P"
    },
    {
      "version": "3.0",
      "score": "9.8",
      "type": "Base",
      "severity": "Critical",
      "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
    }
  ],
  "date": "2020-06-30T17:52+09:00",
  "issued": "2020-06-30T17:52+09:00",
  "modified": "2020-06-30T17:52+09:00"
}
